src/android/java/io/jxcore/node/BtConnectorHelper.java

```diff
@@ -49,18 +55,20 @@ public BtConnectorHelper() {
         this.context = jxcore.activity.getBaseContext();
     }
 
-
-    public BTConnector.WifiBtStatus Start(String peerName,int port){
+    public synchronized BTConnector.WifiBtStatus Start(String peerName,int port){
```

Comment:
Does it ever make sense for start to be called twice on the same BtConnectorHelper object? If not then I would suggest that we can get rid of synchronized and instead just use AtomicBoolean to stick in a guard and call it a day.